### PR TITLE
CLOUDP-67927: mongocli iam organizations|orgs apiKey(s)|apikeys create

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-67927

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```

./bin/mongocli iam org apikeys create --desc "test from mongocli"  --role ORG_OWNER --role ORG_GROUP_CREATOR
{
  "id": "5f22ecea4dca1e5380760a57",
  "desc": "test from mongocli",
  "roles": [
    {
      "orgId": "5e429e7706822c6eac4d5970",
      "roleName": "ORG_GROUP_CREATOR"
    },
    {
      "orgId": "5e429e7706822c6eac4d5970",
      "roleName": "ORG_OWNER"
    }
  ],
  "privateKey": "9004cfb9-d069-47b2-b4ed-43801a8a19e4",
  "publicKey": "xjdbqwmy"
}
```

```
./bin/mongocli iam org apikeys create --desc "test from mongocli"  --role ORG_OWNER --role ORG_GROUP_CREATOR -o ""
APIKey '5f22ed044dca1e5380760a7d' created.
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
Waiting for #317

File: internal/cli/organizations/apikeys/create.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.init()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringSliceVar(&opts.roles, flag.Role, []string{}, usage.Roles)
+	cmd.Flags().StringVar(&opts.desc, flag.Description, "", usage.Comment)
```
Comment (gssbzn) [inline internal/cli/organizations/apikeys/create.go:76]: Isn't description mandatory?

File: internal/description/description.go [hunk 2]
```diff
@@ -117,6 +117,7 @@ const (
 	ListOrganizationAPIKeys     = "List all organization API Keys for a given organization."
 	DescribeOrganizationsAPIKey = "Get a specific Organization API Key."
 	UpdateOrganizationsAPIKey   = "Update an Organization API Key."
+	CreateOrganizationsAPIKey   = "Create an Organization API Key for the specified organization."
```
Comment (gssbzn) [inline internal/description/description.go:120]: [nit] a bit redundant here
Either do `Create an API Key for the specified organization.`
Or `Create an Organization API Key.`

File: internal/cli/organizations/apikeys/create.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	return output.Print(config.Default(), createTemplate, p)
+}
+
+// mongocli iam organizations|orgs apiKey(s)|apikeys create [--role role][--desc description][--orgId orgId]
+func CreateBuilder() *cobra.Command {
+	opts := new(CreateOpts)
+	cmd := &cobra.Command{
+		Use:   "create <name>",
```
Comment (gssbzn) [inline internal/cli/organizations/apikeys/create.go:65]: copy/paste?

File: internal/cli/organizations/apikeys/create.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/organizations/apikeys/create.go:76]: no it is not 🤷‍♂️  https://docs.atlas.mongodb.com/reference/api/apiKeys-orgs-create-one/#request-body-parameters

Comment (gssbzn) [reply internal/cli/organizations/apikeys/create.go:76]: 🤷‍♂️

**Pull request merged**